howso/types.amlg

```diff
@@ -1883,5 +1888,17 @@
 				enum ["string" "list" "assoc" "number" "null" "boolean" "any"]
 				description "The string name of the type a value should be."
 			)
+		TraineeNamePath
```

Comment:
We are not using `name` in context with path since it would be confusing with the actual `name` field. Instead `TraineePath` and `TraineePathLabel` might be better.